src/main/java/org/humancellatlas/ingest/core/service/MetadataCrudService.java

```diff
@@ -0,0 +1,44 @@
+package org.humancellatlas.ingest.core.service;
+
+import lombok.AllArgsConstructor;
+import lombok.NonNull;
+import org.humancellatlas.ingest.core.EntityType;
+import org.humancellatlas.ingest.core.MetadataDocument;
+import org.humancellatlas.ingest.core.Uuid;
+import org.humancellatlas.ingest.core.service.strategy.MetadataCrudStrategy;
+import org.humancellatlas.ingest.core.service.strategy.impl.*;
+import org.humancellatlas.ingest.submission.SubmissionEnvelope;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class MetadataCrudService {
```

Comment:
Why "Crud"?

Reply:
Create/Read/Update/Delete

Reply:
I meant, it seems that it only handles "create"

Reply:
For now but it's purposed is to expose various crud methods that are generic to MetadataDocuments(like addToSubmissionEnvelope). 

This line could be moved to here: https://github.com/HumanCellAtlas/ingest-core/pull/56/files/0458ae1e475dde67e904609142bfbcd22363a3ec#diff-fc7aa7c3c6ceadfd3f41677640c8a3a9R22

Same with lines 23-26 in ValidationStateChangeService. I'll do that refactoring as part of a commit in this pr